fragment-args-ktx/src/main/kotlin/FragmentArgsDelegates.kt

```diff
@@ -620,7 +638,10 @@ public fun <T : Serializable> Bundle?.serializableListNullable(
  * If the key is `null`, uses name of the property as the key.
  * Returns result of [default] function if there is no argument for the given key. Default value is `0`.
```

Comment:
The default values is still here, hiding in the docs here and in other methods :)